chronos/__init__.py

```diff
@@ -185,6 +203,13 @@ def _check_fields(self, job):
             if k not in job:
                 raise MissingFieldError("missing required field %s" % k)
 
+        # Chronos v3.x rejects job names with spaces in them
+        regex = re.compile(r'^[\w.-]+$')
```

Comment:
what about < 3 though? this isn't really allowing for backwards compatibility - if I were you I'd let this happen and have clients deal with the failure (hopefully chronos spits out a reasonably informative error message?)